demos/src/Experiments/Figure/Vue/figure.ts

```diff
@@ -105,7 +105,7 @@ export const Figure = Node.create<FigureOptions>({
           // set cursor at end of caption field
           .command(({ tr, commands }) => {
             const { doc, selection } = tr
-            const position = doc.resolve(selection.to - 2).end()
+            const position = doc.resolve(selection.to).end()
```

Comment:
I don't know what's going on here. Both `selection.to` and `selection.to - 1` work here but with `selection.to - 1`, when I add the figure at the beginning of the document, it doesn't automatically focus on the `figcaption` but with `selection.to` it does.